src/commonMain/kotlin/de/voize/semver4k/Semver.kt

```diff
@@ -470,6 +470,23 @@ class Semver @JvmOverloads constructor(val originalValue: String, val type: Semv
         return with(major!!, minor!!, patch!! + 1, false, false)
     }
 
+    fun nextIncrement(): Semver {
+        if (suffixTokens == null || suffixTokens.isEmpty()) {
+            if (patch != null) {
+                return withIncPatch()
+            }
+
+            if (minor != null) {
+                return create(type, major!!, minor!!.plus(1), null, null, build);
+            }
+                return create(type, major!!.plus(1), null, null, null, build);
+
+        }
+
+        val suffixes = suffixTokens.joinToString(separator = ".")
+        return withSuffix(suffixes + ".0")
+    }
```

Comment:
can you add test cases for nextIncrement

Reply:
tests added